azure-kusto-data/tests/test_streaming_query.py

```diff
@@ -213,10 +213,10 @@ async def test_exception_in_row_async(self):
 
 class TestJsonTokenReader:
     def get_reader(self, data) -> JsonTokenReader:
-        return JsonTokenReader(StringIO(data))
+        return JsonTokenReader(BytesIO(data.encode("utf-8")))
```

Comment:
nit: I think UTF-8 is the default anyway

Reply:
Thanks